lib/features/navigator/utils/navigation_commands.dart

```diff
@@ -0,0 +1,47 @@
+import "package:flutter_riverpod/flutter_riverpod.dart";
```

Comment:
Initially I was wondering "from where the `ref.navigateHome()` come from" extremely useful extension here.

Reply:
Yeah, I've been playing around with different approaches and this in one place was usefull when making any changes